sdk/docs/manually-written/overview/overview/index.rst

```diff
@@ -91,6 +94,10 @@ Continue diving into the elements of the :brokenref:`Canton Network <overview_ov
 
 .. we need to add more references to the protocol and the global sync.
 
+.. figure:: arch-intro-2b.png
```

Comment:
1. We normally use the `.. image::` directive and SVGs.
2. Please add a URL to the source of the diagram as `..` comment before the image so that we can find it again.